home/forms/professor_forms.py

```diff
@@ -203,6 +203,10 @@ def send_webhook(self):
         embed = DiscordEmbed(title=title,
             url=self.request.build_absolute_uri(reverse("admin")))
         webhook.add_embed(embed)
+
+        if num % config.WEBHOOK_FREQUENCY != 0:
+            return
+
         webhook.execute()
```

Comment:
I don't think this method (`ProfessorForm.send_webhook`) is ever called...? should be removed in favor of the utils method if so